ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { test, expect } from '@playwright/test';
+import {
+  AtomButton,
+  clickInTheMiddleOfTheScreen,
+  pressButton,
+  takeEditorScreenshot,
+  waitForLoad,
+  openFileAndAddToCanvas,
+  openPasteFromClipboard,
+  waitForPageInit,
+  selectAtomInToolbar,
+  selectLeftPanelButton,
+  dragMouseTo,
+  LeftPanelButton,
+  selectDropdownTool,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  moveMouseToTheMiddleOfTheScreen,
+  resetCurrentTool,
+  receiveFileComparisonData,
+  saveToFile,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+test.describe('load as fragment (Add to Canvas) srtuctures from files with different formats', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - mol-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1854
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('Molfiles-V2000/cyclopentyl.mol', page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - rxn-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1860
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/cyclopentyl.mol', page);
+    // Coordinates for rectangle selection
+    const startX = 525;
+    const startY = 260;
+    const endX = 765;
+    const endY = 460;
```

Comment:
Perhaps we should put all the constants at the beginning of the test?

Reply:
Done